tests/regression/26-undefined_behavior/09-nullpointer-dereference-struct.c

```diff
@@ -0,0 +1,21 @@
+// PARAM: --enable ana.nullptr --enable dbg.debug
```

Comment:
Don't regression tests have `dbg.debug` universally enabled?

Reply:
for me they fail if I dont have this enabled

Reply:
Hmm, apparently it depends on some weird conditions in the regression testing script:
https://github.com/goblint/analyzer/blob/3690e97b968298cf7854fe12875647743bf2428b/scripts/update_suite.rb#L139-L186

Namely, if the program contains an `assert` anywhere in it, then `dbg.debug` is enabled. Most tests contain some assertions so that's why it seemed to me like that's universally the case.

I'm not sure why this needs to be so complicated, if it's up to me, I'd just make the script always enable it, but maybe @michael-schwarz has a different thought?

Reply:
> I'd just make the script always enable it, but maybe @michael-schwarz has a different thought?
No, I agree!

Reply:
alright, fixed it for all regression tests!